tree/tree/test/ElementStruct.h

```diff
@@ -6,11 +6,8 @@
  * inputs to the test cases.
  */
 
-class ElementStruct : public TObject {
-
+class ElementStruct {
```

Comment:
This commit should be fused with the previous one.

Reply:
Done.

Reply:
Note: it is better/easier for the reviewer if the fusing is done after the next round of review, otherwise knowing what was changed/fixed become an exercise in memorizing code snippet ....